src/components/SwipeClassifier.js

```diff
@@ -73,6 +89,8 @@ export class SwipeClassifier extends React.Component {
     const renderClassifierOrTutorial = () => {
       const key = this.props.workflow.first_task //always just one task
       const task = this.props.workflow.tasks[key]
+      const answers = reverse(task.answers)  //Yes is listed first in project, but we need No listed first (on left)
```

Comment:
Ok, good. Just wanted to make sure this was clear somewhere to the project creator.